src/IsolatedExternalsPlugin.ts

```diff
@@ -47,12 +47,18 @@ function wrapApp(
 ): ConcatSource {
   const externalsList = Object.entries(externals);
   const varNames = externalsList
-    .map(([, external]) => `var ${external.name} = context.${external.name};`)
+    .map(
+      ([, external]) =>
+        `var ${external.name} = context.${external.name} || ( window || global || self )["${external.name}"];`
```

Comment:
It's just a fallback if a dependency is behaving badly and puts their global var on the window in spite of our efforts. That might be a larger problem that people will have to deal with since it's polluting their global space, but we can at least fall back to the window here.

That does bring up a good question, though: do we want this to try to help in that way, or do we want it to break if the global vars don't end up on the context?

Reply:
I think I'm fine with it. Especially as we are the primary users. It will definitely be something to keep an eye on